packages/frontend/components/composed/journal/JournalHero/JournalHero.styles.ts

```diff
@@ -1,6 +1,30 @@
 import styled from "styled-components";
+import { pxToRem } from "@wdp/lib/theme/functions";
 
 export const HeroInner = styled.div`
   padding-block-start: var(--container-padding-lg);
   padding-block-end: var(--container-padding-lg);
 `;
+
+export const TitleBlock = styled.div`
+  display: flex;
+  flex-direction: column;
```

Comment:
Let's not use flex unless we have to - A div aligns it's child items in a column by default. An exception is made if we're modifying the child item's size or alignment using flex.